src/config/price-provider-assets.ts

```diff
@@ -279,13 +285,15 @@ const defaultNetworkData = {
       asset: 'dot',
       address: '0x93aB601aCa2758793a604DDb867BAc4FD91786Da',
     },
+    //usdt
```

Comment:
Why don't we just use USDC for both?

Reply:
i wanted to keep TRN=USDT and BRN=USDC to have at least a couple of cents difference between them since they’re different tokens
but we can use same if that makes more sense